public/meta.ejs

```diff
@@ -0,0 +1,10 @@
+<meta charset="UTF-8" />
```

Comment:
EJS files can't be in public

Reply:
> EJS files can't be in public

@aravindvnair99 Done, fixed accesslint's issuses too